swift/ql/lib/codeql/swift/controlflow/internal/PrintCFG.ql

```diff
@@ -7,52 +7,36 @@
  * @tags ide-contextual-queries/print-cfg
  */
 
-private import codeql.IDEContextual
+private import codeql.swift.elements.File
 private import codeql.swift.controlflow.ControlFlowGraph
-private import codeql.swift.controlflow.internal.ControlFlowGraphImpl::TestOutput
+private import codeql.swift.controlflow.internal.ControlFlowGraphImpl as Impl
+private import codeql.swift.controlflow.internal.ControlFlowGraphImplSpecific
 
-/**
- * Gets the source file to generate a CFG from.
- */
 external string selectedSourceFile();
```

Comment:
Why have the comments on `selectedSourceFile`, `selectedSourceLine` and `selectedSourceColumn` been deleted?